components/shared_data_model/src/shared_data_model/sources/azure_devops.py

```diff
@@ -1,24 +1,23 @@
 """Azure DevOps source."""
 
-from ..meta.entity import Color, EntityAttributeType
-from ..meta.source import Source
-from ..parameters import (
+from shared_data_model.meta.entity import Color, EntityAttributeType
+from shared_data_model.meta.source import Source
+from shared_data_model.parameters import (
+    URL,
```

Comment:
Weird sorting order; do capital letters take precedence, or is it entirely-capsed objects because they are seen as constants?